schemas/uptime-configs.v1.schema.json

```diff
@@ -29,6 +29,22 @@
         "url": {
           "description": "The actual HTTP URL to check.",
           "type": "string"
+        },
+        "request_method": {
+          "description": "The HTTP method to use for the request.",
+          "type": "string",
+          "enum": ["GET", "POST", "HEAD", "PUT", "DELETE", "PATCH", "OPTIONS"]
+        },
+        "request_headers": {
+          "description": "Additional HTTP headers to send with the request",
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          }
+        },
+        "request_body": {
+          "description": "Additional HTTP headers to send with the request",
+          "type": "string"
```

Comment:
Now that I'm thinking about it... should the body also just be a json dict? Mostly people are sending specific key/val params here right?

Reply:
no I think it's very reasonable someone might want to pass xml or some other format